colabs/Zbiory_treningowe_i_testowe.ipynb

```diff
@@ -399,90 +428,314 @@
+            "|-111.8125|25.5625| 199.0206|300.57986|137.95583| -86.48953|2.7391145|51.853218| -1.8940843|0.0|  0.0|       0.0|2.9350004|       0.0|0.074400686|0.0|286.96497|41.560482|0.0|      0.0|     0.0|   287.15115|    287.85394|     288.62442|      289.70667|   10.215544|    46.368584|      97.20809|       178.4421|    153.79295|     332.2353|153.79271|   10.215728|    46.368805|     97.208595|      178.44218|      23.604206|      37.842762|160.57205|0.24502684|2.8677413| 1.3258069|    0.0|0.0028350812| 0.014155917|9.2448574E-4|0.24627747| 0.8884765| 0.8746007| 0.2572757|225.00002|0.009264804|4.5096517|0.14412326|      NULL|2000|    1|        1|\n",
+            "|-111.6875|25.5625| 199.5201|297.45187| 138.2812| -87.80175|3.7463727|49.774567| -1.7860214|0.0|  0.0|       0.0|4.0108986|       0.0|        0.0|0.0|  286.622| 41.56183|0.0|      0.0|     0.0|   286.80237|     287.4586|     288.23932|      289.41895|   10.261331|    41.215076|     80.528015|     105.003525|    132.00487|    237.00838| 237.0086|   10.261422|    41.215225|      80.52848|     105.003525|      14.152389|      26.999157|185.52579|0.29973117|3.9506721| 1.5716397|    0.0| 0.003418415| 0.016775561|0.0012744799|0.29071805| 0.8812726| 0.8061598|0.15693596|     70.0|0.009735265|3.2686296|0.14412326|      NULL|2000|    1|        1|\n",
+            "+---------+-------+---------+---------+---------+----------+---------+---------+-----------+---+-----+----------+---------+----------+-----------+---+---------+---------+---+---------+--------+------------+-------------+--------------+---------------+------------+-------------+--------------+---------------+-------------+-------------+---------+------------+-------------+--------------+---------------+---------------+---------------+---------+----------+---------+----------+-------+------------+------------+------------+----------+----------+----------+----------+---------+-----------+---------+----------+----------+----+-----+---------+\n",
+            "only showing top 20 rows\n",
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%time\n",
+        "size_test2 = splitted_dataset.filter('TEST_ITEM = 1').count()\n",
+        "size_train2 = splitted_dataset.filter('TEST_ITEM = 0').count()\n",
+        "print(f'We have {size_train2} records in train - which is {size_train2/(size_train2+size_test2)*100:3.0f}%')\n",
```

Comment:
W sumie, to jeśli bardzo zwracamy uwagę na czas, to chyba nie musimy tutaj robić tych countów, bo te wartości są nam znane: liczba wszystkich wierzcholków*liczba miesiecy - np. dla zbioru testowego bedzie to 21*76 360 (bo bierzemy ostatnie 2 lata, przy czym z 2023 roku mamy obserwacje tylko z 9 miesiecy)?                                                                                 (Dwa country sumarycznie zajmują ok. 6 minut -  po 3 minuty każdy)

Reply:
Co się spodziewamy a co mamy to czesto pole do zaskoczeń. Na pierwszy rzut oka liczymy tylko proporcję, ale na drugi widzimy czy nasze zapytania z grubsza dobrze działąją.